### PR TITLE
Added SolidAngle and ThermalConductivity.

### PR DESCRIPTION
**Pull request by baxcoders**
SolidAngle - Steradian (sr)
ThermalConductivity - WattPerMeterKelvin (W/m·K)

### CODE REVIEW

File: UnitsNet/UnitDefinitions/SolidAngle.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  "Name": "SolidAngle",
+  "BaseUnit": "Steradian",
+  "XmlDoc": "In geometry, a solid angle is the two-dimensional angle in three-dimensional space that an object subtends at a point.",
+  "XmlDocRemarks": "https://en.wikipedia.org/wiki/Solid_angle",
+  "Units": [
+    {
+      "SingularName": "Steradian",
+      "PluralName": "Steradians",
+      "FromUnitToBaseFunc": "x",
+      "FromBaseToUnitFunc": "x",
```
Comment (angularsen) [inline UnitsNet/UnitDefinitions/SolidAngle.json]: Consider adding these units?
http://www.calculator.org/property.aspx?name=solid+angle

File: UnitsNet/UnitDefinitions/ThermalConductivity.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+﻿{
+  "Name": "ThermalConductivity",
+  "BaseUnit": "WattPerMeterKelvin",
+  "XmlDoc": "Thermal conductivity is the property of a material to conduct heat.",
+  "XmlDocRemarks": "https://en.wikipedia.org/wiki/Thermal_Conductivity",
+  "Units": [
```
Comment (angularsen) [inline UnitsNet/UnitDefinitions/ThermalConductivity.json]: Any prefix units to add?
Add these units?
http://www.unitconversion.org/unit_converter/thermal-conductivity.html

File: UnitsNet/UnitDefinitions/SolidAngle.json (hunk 1 above)
Comment (baxcoders) [reply UnitsNet/UnitDefinitions/SolidAngle.json]: We listed in this PR only the units we're really using. We know that probably their relative physical quantities could be expanded with more definitions and prefixes but they're out of scope at the moment.
We are committed to adding them in future if their usage will become necessary.

Comment (angularsen) [reply UnitsNet/UnitDefinitions/SolidAngle.json]: That is fair. They can be added later if someone finds them useful.

**Pull request merged**